keep/providers/graylog_provider/graylog_provider.py

```diff
@@ -0,0 +1,613 @@
+                        "value": keep_api_url,
+                        "type": "literal",
+                    }
+                )
+                self.__update_url_whitelist(url_whitelist)
+
+            # Create notification
+            notification = self.__get_notification(
+                page=1, per_page=1, notification_name=notification_name
+            )
+            if int(notification["count"]) > 0:
+                self.logger.info("Notification already exists, deleting it")
+                self.__delete_notification(
+                    notification_id=notification["notifications"][0]["id"]
+                )
+
+            self.logger.info("Creating new notification")
+            notification_body = {
+                "title": notification_name,
+                "description": "Hello, this Notification is created by **KeepHQ**, please do not change the title.",
+                "config": {
+                    "type": "http-notification-v2",
+                    "basic_auth": None,
+                    "api_key_as_header": False,
+                    "api_key": "",
+                    "api_secret": None,
+                    "url": keep_api_url,
+                    "skip_tls_verification": True,
+                    "method": "POST",
+                    "time_zone": "UTC",
+                    "content_type": "JSON",
+                    "headers": f"X-API-KEY:{api_key}",
+                    "body_template": "",
+                },
+            }
+            new_notification = self.__create_notification(
+                notification_name=notification_name, notification_body=notification_body
+            )
+
+            for event_definition in event_definitions:
+                if event_definition["_scope"] == "SYSTEM_NOTIFICATION_EVENT":
+                    self.logger.info("Skipping SYSTEM_NOTIFICATION_EVENT")
+                    continue
+                self.logger.info(f"Updating event with ID: {event_definition['id']}")
+                event_definition["notifications"].append(
+                    {"notification_id": new_notification["id"]}
+                )
+                self.__update_event(event=event_definition)
+
+            self.logger.info("Webhook setup completed successfully")
+        except Exception as e:
+            self.logger.error(
+                "Error while setting up webhook", extra={"exception": str(e)}
+            )
+            raise e
+
+    @staticmethod
+    def __map_event_to_alert(event: dict) -> AlertDto:
+        return AlertDto(
+            id=event["event"]["id"],
+            name=event.get("event_definition_title", event["event"]["message"]),
+            severity=[AlertSeverity.LOW, AlertSeverity.WARNING, AlertSeverity.HIGH][
+                int(event["event"]["priority"]) - 1
+            ],
+            description=event.get("event_definition_description", None),
+            fingerprint=event["event"]["event_definition_id"],
```

Comment:
can you use the convention of FINGERPRINT_FIELDS?

see https://github.com/keephq/keep/blob/519ab28c6fd9c3f12f2aab578e7d065007c4d2ce/keep/providers/gcpmonitoring_provider/gcpmonitoring_provider.py#L51